vars/neoDeploy.groovy

```diff
@@ -186,6 +193,26 @@ def call(parameters = [:]) {
                           dockerEnvVars: configuration.get('dockerEnvVars'),
                           dockerOptions: configuration.get('dockerOptions')) {
 
+                JAVA_HOME_CHECK : {
```

Comment:
The version of Java must be checked also if it is in the path.

Reply:
Not checking the version is not a dis-improvement compared to the situation before with having the check in the pipeline. In case there was the validation before in the pipeline we failed since the path was not taken into account at all. In case we there was no validation before in a pipeline there was no validation at all.

I agree, we should also have a validation in case we have the executable from the path, but this would be done IMO in another pull request, also in order to keep the focus of the PR clear. I created issue #102 .